app/src/main/java/android/example/courses/ScheduleFragment.kt

```diff
@@ -0,0 +1,175 @@
+package android.example.courses
+
+import android.content.Context
+import android.content.res.Resources
+import android.graphics.Color
+import android.os.Bundle
+import android.view.Gravity
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.LinearLayout
+import android.widget.TextView
+
+class ScheduleFragment : Fragment() {
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        val container = inflater.inflate(R.layout.fragment_schedule, container, false)
+        val globalLinearLayout = container.findViewById<LinearLayout>(R.id.globalScheduleFrameLayout)
+        buildPage(globalLinearLayout)
+        return container
+    }
+
+    private fun buildPage(container: LinearLayout) {
+        val day = "Monday"
+        val totalHours = "5.34 hours"
+
+        val layout = LinearLayout(context)
+
+        val params = LinearLayout.LayoutParams(
```

Comment:
I see you experimented with adding views programmatically. And also seen what a pain in the a** it this :)

Reply:
@radumarin88 Indeed, that was an experiment where I just wanted to build out a new fragment programatically and I found out that it's quite difficult to build it, mostly because you need to run it every time to see the effect of your changes. At the end of the day, however, I think it was pretty helpful: I got a better understanding of layouts and view arrangements and how to work with them programatically